.changeset/unlucky-ducks-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
```

Comment:
Not sure what is appropriate for removing a broken command here.

Reply:
Let's keep this for now, since it was actually completely broken and won't hurt anybody in practice